app/lib/experiments.coffee

```diff
@@ -2,25 +2,27 @@ $ = require('jqueryify')
 ExperimentalSubject = require 'models/experimental_subject'
 
 module.exports = class ExperimentServerClient
+  constructor: (@Geordi) ->
 
   # CONSTANTS #
-
   ###
   Define the active experiment here by using a string which exists in http://experiments.zooniverse.org/active_experiments
   If no experiments should be running right now, set this to null, false or ""
   ###
-  ACTIVE_EXPERIMENT:null
+  ACTIVE_EXPERIMENT: "SerengetiMessagingExperiment3"
```

Comment:
Currently we have SerengetiMessagingExperiment2 active (despite it showing null on the left). This just initiates a new run of the experiment.